.droneci/config.yml

```diff
@@ -0,0 +1,33 @@
+kind: pipeline
+name: default
+
+steps:
+- name: build
+  image: ivigns/nice-ubuntu:0.0.1
+  commands:
+  - git submodule sync
+  - git submodule update --init --recursive
+  - git submodule update --recursive
+  - cd /drone/src/sdsl-lite
+  - bash uninstall.sh /usr
```

Comment:
are you sure you need this?

Reply:
I agree that it's an overkill. This was made in case CI reuse the last docker container. I decided if it does nothing bad, I will leave it.